### PR TITLE
Documentation/glossary section

### LINKED ISSUE #1429: [Documentation] Write "Glossary"-section for contribution guidelines
**This is part of a larger issue - please read: https://github.com/kirbydesign/designsystem/issues/1393#issuecomment-802766408 before continuing!**

**Short description**

A "glossary" section should be added to the contribution guidelines.  An outline already exists on the branch [`documentation/contribution-guidelines`](https://github.com/kirbydesign/designsystem/tree/documentation/contribution-guidelines) which should be expanded upon.  

**Edit )** Might be made redundant by #1464

### PR DESCRIPTION
**Pull request by jkaltoft**
This PR closes #1429

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Enhancement (to existing content)
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] cookbook application / infrastructure changes
- [ ] Other... Please describe:

## Which issue documents the current behaviour?

Issue Number: #1429

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ If you at any point have any questions please read: [I have a question related t
 **Beware:** We reserve the right to close issues and pull-requests that does not follow the steps outlined in [the contribution model](#the-contribution-model). This is not because we want to be mean or micro-manage the way things are done - but a project of this size requires some consistency in the way we cooperate in order to be manageable. 
 
 - [Introduction to the contribution model](#introduction-to-the-contribution-model)
+  - [The process of contributing](#the-process-of-contributing)
```
Comment (MadsBuchmann) [inline .github/CONTRIBUTING.md:9]: We should add: 
```- [Before you get coding](#before-you-get-coding)```

after: 
` - [The process of contributing](#the-process-of-contributing)`

**Review by MadsBuchmann — APPROVED** [review]
Looking good! - only had two minor things. 

Adding an additional heading and mixing in some additional formatting with the block quotes, helped a bit on the "wall of text"-feeling. Eventho it is still quite a bit of a wall. 

However I think it is good as is and nothing is set in stone. So if it turns out, that there is an issue with how it is now, we can always change it 😄

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -48,39 +49,62 @@ To do so a series of steps has been created that an issue must go through. All o
...
+#### Tech Refinement
+
+> Discussion and agreement on technical solution(s) for the implementation of the issue.
+
+Every issue must be tech refined. Here it is discussed and agreed on how an issue should be implemented the best; the agreed upon solution should be documented in a comment on the issue. How long it takes depends on the scope of the issue. For small issues it might be nothing more than an agreement via a direct message. Other times it is a full-blown meeting with multiple participants.
+
+Once the issue has been properly tech refined Team Kirby will remove the `NOT Tech refined` label from the issue. _No code should be written before this step is complete_. If a pull-request is submitted for an issue that has not been tech refined; changes and problems that could have been tackled from the beginning, might arise from the code-review.
```
[the remark this answers is not in the document]
Comment (jkaltoft) [reply .github/CONTRIBUTING.md:85]: Agree.

[the remark this answers is not in the document]
Comment (jkaltoft) [reply .github/CONTRIBUTING.md:85]: Is this too much?

image

[the remark this answers is not in the document]
Comment (MadsBuchmann) [reply .github/CONTRIBUTING.md]: Looking good!

**Pull request merged**